src/logic/tokens/store/actions/fetchSafeTokens.ts

```diff
@@ -80,10 +105,7 @@ const fetchSafeTokens = (safeAddress: string) => async (
     const updateActiveTokens = activeTokens.equals(alreadyActiveTokens) ? noFunc : update({ activeTokens })
     const updateBalances = balances.equals(safeBalances) ? noFunc : update({ balances })
     const updateEthBalance = ethBalance === currentEthBalance ? noFunc : update({ ethBalance })
```

Comment:
updating safe's properties in batch like this makes no sense because we're performing updates on the same instance. This is like sending 10 PUT requests to modify 10 properties on the same resource instead of sending one request

Reply:
Also I'm not sure if `.equals` performs checks for deep equality

Reply:
Ok, but we should aim to fix this in a different issue. All the changes in this file were related to typing fixes.

Reply:
Could you please create one?

Reply:
#1185